internal/services/managed_environment/resource_managed_environment_test.go

```diff
@@ -40,6 +40,7 @@ func TestAccManagedEnvironmentsResource_Validate_Create(t *testing.T) {
 					max_limit_user_sharing     = 10
```

Comment:
I fixed unit test a bit, but you still have to test then and fix some of them. Be also sure that all managed environment acceptance test are working.